x/delegation/keeper/grpc_query.go

```
@@ -68,3 +68,15 @@ func (k Keeper) QueryAssociatedStakersByOperator(ctx context.Context, req *deleg
 		Stakers: stakers,
 	}, nil
 }
+
+func (k Keeper) QueryDelegatedStakersByOperator(ctx context.Context, req *delegationtype.QueryDelegatedStakersByOperatorReq) (*delegationtype.QueryDelegatedStakersByOperatorResponse, error) {
+	c := sdk.UnwrapSDKContext(ctx)
+	stakers, err := k.GetStakersByOperator(c, req.Operator, req.AssetId)
+	if err != nil {
+		return nil, err
+	}
+	return &delegationtype.QueryDelegatedStakersByOperatorResponse{
+		Count:   uint32(len(stakers.Stakers)),
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int -> uint32

(gosec)